docs/local.md

```diff
@@ -21,6 +21,11 @@
     brew install gnu-sed
     ```
 
+- If you are on a Mac, install updated `make` with Homebrew (version 4.3 is required)
+    ```shell script
+    brew install make
```

Comment:
Not sure this would be enough:
```
brew install make
...
==> Caveats
GNU "make" has been installed as "gmake".
If you need to use it as "make", you can add a "gnubin" directory
to your PATH from your bashrc like:

    PATH="/usr/local/opt/make/libexec/gnubin:$PATH"
==> Summary
🍺  /usr/local/Cellar/make/4.3: 15 files, 1MB
```